lastminutewf2020.yml

```diff
@@ -57,6 +56,13 @@
         owner: root
         group: root
         mode: '0644'
+    - name: Ensure printsrv in /etc/hosts
+      ansible.builtin.lineinfile:
+        path: /etc/hosts
+        line: "{{ baseip }}.211	printserv"
```

Comment:
```suggestion
        line: "{{ baseip }}.211	printserver"
```

Reply:
actually it should be printsrv,  i have removed that extra e